config/index.js

```diff
@@ -47,6 +47,27 @@ const config = {
     },
     geekJokesApi: {
       urlApi: process.env.API_RANDOM_WEETS
+    },
+    validationSchemas: {
+      user: {
+        name: {
+          minLength: 3,
+          maxLength: 30
+        },
+        lastName: {
+          minLength: 3,
+          maxLength: 30
+        }
+      },
+      credential: {
+        email: {
+          woloxDomainRegex: /@wolox.com?/i
+        },
+        password: {
+          minLength: 8,
+          maxLength: 15
+        }
+      }
```

Comment:
This is not bad but in the end, it seems to me that it can go without problems in the schemas, we better leave this here for the higher-level configuration of the project. Excuse me, I know you showed me before.

Reply:
No problem, I'll move validation constants to the same schemas